system-variables.md

```diff
@@ -6304,7 +6304,7 @@ Internally, the TiDB parser transforms the `SET TRANSACTION ISOLATION LEVEL [REA
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
 - Type: Integer
 - Default value: `8`
-- Range: `[0, 2147483647]` for TiDB Self-Hosted and [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `[8, 2147483647]` for [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless)
+- Range: `[0, 2147483647]` for TiDB Self-Managed and [TiDB Cloud Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `[8, 2147483647]` for [TiDB Cloud Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless)
```

Comment:
Please search `https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated` and replace all instances of it with `https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-cloud-dedicated`

Reply:
Done in https://github.com/pingcap/docs/pull/18744/commits/98fc5d6aa18e51c8844b1c02249e9ab4542ccbce.